stories/molecules/Dropdown.stories.tsx

```diff
@@ -211,69 +149,75 @@ export const _DropdownWithMultiSingleSelectItem: Story = {
-                            key={value}
-                            value={value}
-                            isSelected={selectedValues.includes(value)}
+                            onCheckedChange={() => {
+                                handleOnChange('all');
+                            }}
+                            key="all"
+                            isSelected={isAllSelected()}
+                            variant="select-all"
                         >
-                            {value}
+                            Select all
                         </MultiSelectItem>
-                    ))}
-                    <MultiSelectItem
-                        key="group2"
-                        value="group2"
-                        isSelected={isGroup2Selected()}
-                        variant="group-title"
-                    >
-                        Items
-                    </MultiSelectItem>
-                    {group2.map((value) => (
+                        {generalValues.map((value) => (
+                            <MultiSelectItem
+                                key={value}
+                                isSelected={selectedValues.includes(value)}
+                                onCheckedChange={() => {
+                                    handleOnChange(value);
+                                }}
+                            >
+                                {value}
+                            </MultiSelectItem>
+                        ))}
                         <MultiSelectItem
-                            key={value}
-                            value={value}
-                            isSelected={selectedValues.includes(value)}
+                            key="group1"
+                            isSelected={isGroup1Selected()}
+                            variant="group-title"
+                            onCheckedChange={() => {
+                                handleOnChange('group1');
+                            }}
                         >
-                            {value}
+                            Options
                         </MultiSelectItem>
-                    ))}
-                </Dropdown>
-            </div>
+                        {group1.map((value) => (
+                            <MultiSelectItem
+                                key={value}
+                                isSelected={selectedValues.includes(value)}
+                                onCheckedChange={() => {
+                                    handleOnChange(value);
+                                }}
+                            >
+                                {value}
+                            </MultiSelectItem>
+                        ))}
+                        <DropdownSub>
+                            <SubContentTrigger>Items →</SubContentTrigger>
+                            <DropdownPortal>
+                                <DropdownSubContent>
```

Comment:
The style of the nested field looks different from the design :) 
<img width="220" alt="Screenshot 2024-10-15 at 14 14 08" src="https://github.com/user-attachments/assets/0a5b9961-c676-438f-a6e8-70f47de03318">

Reply:
The style still appears incorrect, and it's implemented directly within the stories. Shouldn't we have a dedicated `NestedItem` component as mentioned in the ticket? Or will that be part of a separate ticket?